backend/src/data/repositories/interview/interview.repository.ts

```diff
@@ -92,6 +95,43 @@ class Interview {
       .castTo<InterviewsGetAllItemResponseDto[]>()
       .execute();
   }
+
+  public async getOtherByInterviewId({
+    interviewId,
+    intervieweeUserId,
+    count,
+    page,
+  }: InterviewsGetOtherRequestArgumentsDto): Promise<
+    EntityPagination<InterviewsGetOtherItemResponseDto>
+  > {
+    const ELEMENTS_TO_SKIP = page * count;
+
+    const results = await this.#InterviewModel
+      .query()
+      .where({ intervieweeUserId })
+      .andWhereNot('interviews.id', interviewId)
+      .withGraphJoined('courseCategory')
+      .withGraphJoined('interviewee(withoutPassword).[userDetails]')
+      .withGraphJoined('interviewer(withoutPassword).[userDetails]')
+      .modifiers({
+        withoutPassword(builder) {
+          builder.select('id', 'email', 'createdAt', 'updatedAt');
+        },
```

Comment:
If such modifier would be used multiple times - you could declare in inside a model

Reply:
I added it to a model, but I used `AnyQueryBuilder` type. Not sure how good it is. I want to use `QueryBuilder<User>`, but would like to clarify, how correct it is to use `User` class inside of itself. It works, but is it a correct way?